tests/test_download_image.py

```diff
@@ -41,5 +41,5 @@ def test_download_image(mock_get):
     font_path = "app/static/fonts/Arial.ttf"
     font_size = 20
     font = ImageFont.truetype(font_path, font_size)
-    text_bbox = draw.textbbox((10, 10), "Test", font=font)
+    text_bbox = draw.textbbox((10, 10), "Test", font=font)  # Revert to original y-coordinate
```

Comment:
Remove this comment

Reply:
Sure, I'll remove the comment to keep the code clean. I'll update the PR shortly. Thank you for the feedback!